src/wp-includes/block-supports/background.php

```diff
@@ -58,9 +58,9 @@ function wp_render_background_support( $block_content, $block ) {
 		return $block_content;
 	}
 
-	$background_image_source = _wp_array_get( $block_attributes, array( 'style', 'background', 'backgroundImage', 'source' ), null );
-	$background_image_url    = _wp_array_get( $block_attributes, array( 'style', 'background', 'backgroundImage', 'url' ), null );
-	$background_size         = _wp_array_get( $block_attributes, array( 'style', 'background', 'backgroundSize' ), 'cover' );
+	$background_image_source = $block_attributes['style']['background']['backgroundImage']['source'] ?? null;
```

Comment:
@jrfnl Are we supporting null coalescing operator in the new coding standards?

Reply:
Have you tried ?

Reply:
> Have you tried ?

What do you mean?

Reply:
Exactly what I'm asking: have you tried it ?

Note in the below workflows the status is 🟢 and that is even on the older WPCS version which is currently still being used...

Reply:
Also related: https://core.trac.wordpress.org/ticket/58874